dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipUdpOutboundChannelFactory.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
```

Comment:
Is this class unimplemented, used, or needed? Everything returns null

Reply:
Is it still trying to fit well with the ChannelFramework patterns (but with no Adapter separated out)?

Reply:
Not needed.  File deleted.